### PR TITLE
Upgrade to canton 0.18.1 in ledger api conformance tests and enabling canton stdout info-level logging

### PR DESCRIPTION
**Pull request by oliverse-da**
changelog_begin
changelog_end

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [X] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [X] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Upgrade all the things!

**Review by SamirTalwar-DA — APPROVED** [review]

File: ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ set -o pipefail
 CANTON_COMMAND=(
   "$(rlocation com_github_digital_asset_daml/ledger/ledger-api-test-tool-on-canton/canton)"
   daemon
+  "-d"
```
Comment (SamirTalwar-DA) [inline ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh:12]: Well, that's nice and easy. 😀

**Comment by oliverse-da** [comment]
fyi @rautenrieth-da

**Pull request merged**